package.json

```diff
@@ -41,7 +41,7 @@
 		]
 	},
 	"lint-staged": {
-		"*": "eslint --fix"
+		"*.js,*.ts": "eslint --fix"
```

Comment:
can you elucidate the value of this?

the thing that gives me slight pause is that it means we'll have to keep it up to date (e.g. if we add `tsx`, `svelte` files etc)

Reply:
It currently runs on other files like JSON and it doesn't know what to do with them. Like so:
![Screenshot 2020-10-12 at 11 48 24](https://user-images.githubusercontent.com/768467/95738297-e8710400-0c80-11eb-89da-7f793b745439.png)

And presumably there is a longer list of extensions to ignore? So figured this was the way forward